lib/sentry/opentelemetry/span_storage.ex

```diff
@@ -0,0 +1,137 @@
+defmodule Sentry.OpenTelemetry.SpanStorage do
+  @moduledoc false
+  use GenServer
+
+  require Logger
+
+  @table :span_storage
+  @cleanup_interval :timer.minutes(5)
```

Comment:
This seems quite long, I think we can shorten the span TTL *and* this? What's the reasoning behind these two numbers?